WooCommerce/Classes/ViewRelated/Upgrades/SubscriptionsView.swift

```diff
@@ -100,10 +100,10 @@ private extension SubscriptionsView {
         static let subscriptionStatus = NSLocalizedString("Subscription Status", comment: "Title for the plan section on the subscriptions view. Uppercased")
         static let experienceFeatures = NSLocalizedString("Experience more of our features and services beyond the app",
                                                     comment: "Title for the features list in the Subscriptions Screen")
-        static let cancelTrial = NSLocalizedString("Cancel Free Trial", comment: "Title for the button to cancel a free trial")
+        static let manageSubscription = NSLocalizedString("Manage your subscription", comment: "Title for the button to manage subscriptions")
```

Comment:
Since this is a button, it should be in Title case: pecCkj-eD-p2
```suggestion
        static let manageSubscription = NSLocalizedString("Manage Your Subscription", comment: "Title for the button to manage subscriptions")
```